newIDE/app/src/UI/TextField.js

```diff
@@ -75,11 +75,12 @@ type Props = {|
   margin?: 'none' | 'normal',
   fullWidth?: boolean,
   style?: {|
-    fontSize?: 18,
+    fontSize?: number | string,
```

Comment:
Can we have `fontSize?: 14 | 18`?

This is to ensure consistency in the interface as much as possible, discouraging new component to do "fancy stuff" with text fields. Trying to reduce the "[codebase entropy](https://www.webopedia.com/TERM/S/software_entropy.html)" in a way 😄